### PR TITLE
Huge change for teams in a database

### PR DESCRIPTION
**Pull request by holyzala**

### CODE REVIEW

File: project/eScavenge/Team.py [hunk 1]
```diff
@@ -1,95 +1,56 @@
...
-        pass
+        return self.__points
 
     @points.setter
-    @abstractmethod
     def points(self, points):
-        pass
+        self.__points = max(0, points)
+
+    def add_penalty(self, penalty=1):
```
Comment (Purlieu) [inline project/eScavenge/Team.py]: Are we assuming now that the penalty function will just be called and the value will always be one?

Comment (holyzala) [reply project/eScavenge/Team.py]: It defaults to one, you can override it on call to whatever you want

Comment (holyzala) [reply project/eScavenge/Team.py]: So add_penalty() will add 1, add_penalty(5) will add 5

Comment (Purlieu) [reply project/eScavenge/Team.py]: Gotcha okay, that makes sense.  I assumed Python always defaulted to that value.

**Pull request merged**